languages/csharp/reference/exercise-concepts/robot-name.md

```diff
@@ -0,0 +1,19 @@
+# Concepts of Robot Name
```

Comment:
Could you add a link to the example implementation? See https://github.com/exercism/v3/pull/364/files#diff-8550dac7ec316a38e48f5543e52ec570R3